event/listener.php

```diff
@@ -111,6 +112,22 @@ public function do_poll_modification($event)
 	}
 
 	/**
+	 * Modifys the template vars to match the advanced poll settings
+	 *
+	 * @param object $event The event object
+	 * @return void
+	 */
+	public function do_poll_block_modification($event)
+	{
+		if ($event['s_display_results'] && (int) $event['topic_data']['wolfsblvt_poll_show_ordered'])
```

Comment:
Okay, in this case we have to sort manually after giving the vote via ajax. Extending the vote function there.
You can create an issue for that, than I will remember it that I have to do it for your stuff.